README.rst

```diff
@@ -41,7 +41,8 @@ Configuration
         'default_project': 'SWCC',
         'relationship_to_parent': ('depends_on', r'MEETING-[\w_]+'),
         'components': '[SW],[HW]',
-        'description_head': 'Action raised in meeting.\n\n',
+        'description_head': 'Action raised in [this meeting|https://docserver.com/<<file_name>>.html].\n\n',
+        'str_to_attr_mapping': {'<<file_name>>': 'docname'},
```

Comment:
`description_str_to_attr` might be better.

Reply:
Definitely! Also, the main description for the Jira ticket can benefit from this feature (not only description_head).

Reply:
Done in ab29d44b28f4b66f702b6f1e34168916480356b6